src/Track/Exporter.php

```diff
@@ -32,7 +32,11 @@ public function exportGpx(iterable $versionCollection): string
 
         foreach ($versionCollection as $version) {
             $trkXml = $xml->addChild('trk');
-            $trkXml->addChild('name', $version->getTrack()->getNameEn());
+
+            $name = $version->getTrack()->getNameEn();
+            if ($name) {
```

Comment:
this should be `if ($name === null)`
because string "0" is `false`

Reply:
Fixed